src/commands/install.ts

```diff
@@ -14,13 +14,13 @@ import {InputOptions, OutputOptions, rollup, RollupError} from 'rollup';
 import validatePackageName from 'validate-npm-package-name';
 import {EnvVarReplacements, SnowpackConfig} from '../config.js';
 import {resolveTargetsFromRemoteCDN} from '../resolve-remote.js';
+import {rollupPluginCatchUnresolved} from '../rollup-plugin-catch-unresolved.js';
```

Comment:
🔠

Reply:
lol I can't take credit for this, there's a great "Organize Imports" command inside of VSCode that automates this :)